### PR TITLE
Feature/gifts command

### PR DESCRIPTION
**Pull request by b-dabrowski**

### CODE REVIEW

File: src/common/services/gift.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import TranslationsService from './translationsService'
+
+export default class GiftService {
+  private translationsService: TranslationsService
+
+  constructor() {
+    this.translationsService = new TranslationsService()
+  }
+
+  public async initGifts() {
```
Comment (TomaszChmielPagepro) [inline src/common/services/gift.ts]: Add TODO comment here

File: src/models/gift.model.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export interface IGift {
+  name: string,
+  description: string,
+  cost: number,
+  isAvailable?: boolean
+}
+
+type IGiftDocument = IGift & Document
+
+const giftSchema: Schema<IGift> = new Schema({
```
Comment (TomaszChmielPagepro) [inline src/models/gift.model.ts]: I think gifts should be assigned to workspace

File: src/controllers/slackController/index.ts [hunk 3]
```diff
@@ -18,9 +18,11 @@ export default class SlackController {
     const { challenge, event } = slackEventInfo
 
     if (challenge) {
-      return res.send(body)
+      res.send(body)
     }
 
+    res.sendStatus(200)
```
Comment (TomaszChmielPagepro) [inline src/controllers/slackController/index.ts]: Why is this moved from bottom? :P

Comment (b-dabrowski) [reply src/controllers/slackController/index.ts]: I found out that it is best practice to answer to event or action as quick as possible and then do business logic so that is why I move res.sendStatus(200)

https://api.slack.com/tutorials/intro-to-message-buttons
